### PR TITLE
Add contribex-comms blog shadow guide

### LINKED ISSUE #7875: Add a shadowing guide for SIG Contribex blogging
### Describe the issue

Since we're taking shadows, it would make sense to have a document that would serve as a guide, identifying what is expected, in a similar way to what exists for e.g. Kubernetes release shadows.

Content could include:

* Typical tasks
* Useful skills
* Time expectations

### PR DESCRIPTION
**Pull request by fsmunoz**
Initial version of the Contribex Comms blogging shadowing guide.

Fixes #7875

### CODE REVIEW

File: communication/contributor-comms/blogging-resources/blogging-shadow-guide.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Contributor Comms Blogging: Shadow Guide
+
+Contribex Comms has leads for different areas, specifically for
+blogging where there's a Blogging Lead that takes additional
+responsibility for making sure that the blogging process not only
+works as expected, but is constantly being improved to better serve the
+goals in the [SIG charter]((./CHARTER.md).
```
Comment (ArvindParekh) [inline communication/contributor-comms/blogging-resources/blogging-shadow-guide.md:7]:
```suggestion
goals in the [SIG charter](./CHARTER.md).
```
Removing the extra bracket to maintain the markdown link format

**Comment by kaslin** [comment]
/lgtm
Would like to see at least one more lgtm before adding an approve label. @ArvindParekh? @chris-short? Other current Blogging Shadows? Would love to hear from current shadows if this content seems helpful to them.

**Comment by kaslin** [comment]
Added a hold because I realized it already has the approved label, since it was created by a lead with approver privileges. Soft requirement on the additional lgtm, but I think it would be good to have.

File: communication/contributor-comms/blogging-resources/blogging-shadow-guide.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+aware of the expected commitment. The main requirement for shadowing
+is a willingness to participate actively and consistently.
+
+While not requirements, the following are things that will be a part
+of the work expected from shadows, so any existing mastery of them is
+certainly a plus:
+
+* Knowledge of git and GitHub (issues, PRs, reviews, merges, etc)
+* Knowledge of the existing blogging standards, including the ones
+  external to SIG Contribex Comms (like the SIG Docs standards)
```
Comment (ArvindParekh) [inline communication/contributor-comms/blogging-resources/blogging-shadow-guide.md:55]:
```suggestion
  external to SIG Contribex Comms (like the [SIG Docs standards](https://kubernetes.io/docs/contribute/style/style-guide/))
```
New contributors might not know about the existing blogging standards. Should we add a link here to the Documentation style guide?
Also, are there any other blogging standards that I'm not aware of?

File: communication/contributor-comms/blogging-resources/blogging-shadow-guide.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+* Participation in the weekly SIG Contribex Comms meetings
+
+In terms of time requirements, this is a very flexible programme since
+writing articles is something that can be done asynchronously. The
+following provides a minimal guideline:
```
Comment (ArvindParekh) [inline communication/contributor-comms/blogging-resources/blogging-shadow-guide.md:60]: Probably a nit, but should we mention that the mentioned time requirements are on a per week basis?

File: communication/contributor-comms/blogging-resources/blogging-shadow-guide.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+* Help contributors: the shadow will reply to contributors in the team
+  Slack channel, steering them in the right direction.
+* Propose a new topic: the shadow will identify new interesting
+  articles and add them to the project board.
+
+The following stage includes tasks that require more direct
+involvement:
+
+* Write an interview script.
+* Open an Issue and a PR.
```
Comment (ArvindParekh) [inline communication/contributor-comms/blogging-resources/blogging-shadow-guide.md:87]: While giving this a read, it sounded a bit unclear. 
```suggestion
* Open an Issue and a PR to publish the completed article.
```

File: communication/contributor-comms/blogging-resources/blogging-shadow-guide.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+* Carry an article from inception to publication, updating the project
+  board and Slack channel.
```
Comment (ArvindParekh) [inline communication/contributor-comms/blogging-resources/blogging-shadow-guide.md:89]: First-timers could misinterpret this as updating the Slack channel in some way. I know we mean keeping the SIG informed about the progress on Slack, and it's probably a nit, but putting it out there to see if it's confusing to someone else too. If not, totally good to go 👍🏻

**Review by ArvindParekh — COMMENTED** [review]
> Would love to hear from current shadows if this content seems helpful to them.

It does! I gave the article a read again as a new contributor and I found some areas that could benefit from a little more information. Would love to hear your thoughts about it.
Just a few suggestions, let me know if they're helpful or not, and I'd be happy to go ahead and add the additional lgtm @kaslin :)

File: communication/contributor-comms/blogging-resources/blogging-shadow-guide.md (hunk 3 above)
Comment (fsmunoz) [reply communication/contributor-comms/blogging-resources/blogging-shadow-guide.md:60]: Added another "weekly" in the second point.

File: communication/contributor-comms/blogging-resources/blogging-shadow-guide.md (hunk 5 above)
Comment (fsmunoz) [reply communication/contributor-comms/blogging-resources/blogging-shadow-guide.md:89]: Updated working to make it clearer.

**Pull request merged**